domino/domino.py

```diff
@@ -425,12 +432,13 @@ def validate_is_external_volume_mounts_supported():
                 "masterHardwareTierId": master_hardware_tier_id
             }
 
+        resolved_hardware_tier_id = hardware_tier_id if hardware_tier_id is not None else self.get_hardware_tier_id_from_name(hardware_tier_name)
```

Comment:
You might be able to shorten this line with...
```suggestion
        resolved_hardware_tier_id = hardware_tier_id or self.get_hardware_tier_id_from_name(hardware_tier_name)
```